sdk/turing/router/config/router_config.py

```diff
@@ -216,20 +220,40 @@ def to_open_api(self) -> OpenApiModel:
             kwargs['enricher'] = self.enricher.to_open_api()
         if self.ensembler is not None:
             kwargs['ensembler'] = self.ensembler.to_open_api()
+            if kwargs['ensembler'] is None:
+                # Nop ensembler, delete
+                del kwargs['ensembler']
```

Comment:
The Turing API does not handle an ensembler type "nop" - it must be left unset.

Reply:
Would it be clearer to add this comment into the code file itself too so that we don't need to dig PRs to figure out why the value needs to be deleted?

Reply:
Done.